src/scripts/extension-utils.js

```diff
@@ -18,6 +18,8 @@ const processMessage = (store, type, payload) => {
     case "GET_WALLETS_RESPONSE":
       store.commit("setExtensionAccounts", payload)
       break
+    default:
+      return
```

Comment:
missing test ;)

Reply:
🤦‍♂

Reply:
I'm still seeing the red line above, but I have a test for it? Does CodeClimate take a little while… or?

Reply:
green line now!